build.gradle

```diff
@@ -28,6 +28,7 @@ dependencies {
     implementation 'io.github.acm19:aws-request-signing-apache-interceptor:2.3.1'
 
     implementation 'com.amazonaws:aws-lambda-java-core:1.2.3'
+    implementation 'com.amazonaws:aws-lambda-java-events:3.7.0'
```

Comment:
The latest one is `[3.11.5](https://mvnrepository.com/artifact/com.amazonaws/aws-lambda-java-events/3.11.5)`, can you please update ?